### DISCUSSION TITLE
Fusion Dark theme, DDL: black on black

### DISCUSSION THREAD

**Discussion by MetallicCrimson** [pawelsalawa/sqlitestudio#4929] [Q&A] [answered]
Hello!  
First of all: I'm *very* new to SQLiteStudio, so sorry, if the answer is obvious.
If I choose the Fusion Dark theme, and try to view the DDL of a table in a DB, it shows black text with a dark gray background.

![image](https://github.com/pawelsalawa/sqlitestudio/assets/67794509/d6107fa9-f896-48a8-9592-b080148e6ea3)

I've found a previous discussion about a similar issue in the SQL editor, but that (as everything else) looks fine here.
Is there a way to change this?

Thank you!

**Comment by pawelsalawa** [comment, marked as answer]
Try this:
![image](https://github.com/pawelsalawa/sqlitestudio/assets/481846/945413ff-ef8a-4c67-972c-e14d56bd97aa)

**Reply by MetallicCrimson** [reply]
Oh wow, that worked, despite that I didn't even touch the individual colors. Awesome, thank you so much!
